### PR TITLE
Enable filter and funnel feature flags

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- WIP, enabling the feature flags for the release and going to fix broken tests

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Comment by mariusandra** [comment]
This is finally ready for a review:
- Added in the `4267-taxonomic-property-filter` and `4535-funnel-bar-viz"` flags.
- Had to change a bunch of cypress tests to make them work with the new property filter
- Limited the new cohort property selection to just person properties & cohorts (removed event properties)
- Slight cleanup to the "no results" text
- Removed the "save-cohort-on-modal" flag and made the button visible only with clickhouse.

One caveat:
- It seems that the cohort breakdown filter also has one special row called "All Users", which is a cohort `{id: 'new'}` that I didn't implement. I'll add it in a new PR and re-enable the test that clicks it. It turned out to be more tricky to include it here.

Except for that test, all the rest is working. Something is wrong with the cypress component tests though, so restarting the run.

File: frontend/src/scenes/trends/personsModalLogic.ts [hunk 1]
```diff
@@ -101,6 +102,10 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
             (s) => [s.peopleLoading, s.loadingMorePeople],
             (peopleLoading, loadingMorePeople) => peopleLoading && !loadingMorePeople,
         ],
+        clickhouseFeaturesEnabled: [
```
Comment (paolodamico) [inline frontend/src/scenes/trends/personsModalLogic.ts:105]: This selector is duplicated from `funnelLogic`, was considering abstracting it, but as it's straightforward it's probably fine.

**Review by paolodamico — APPROVED** [review]
lgtm! hope fixing all these tests wasn't too time consuming

File: frontend/src/scenes/trends/personsModalLogic.ts (hunk 1 above)
Comment (mariusandra) [reply frontend/src/scenes/trends/personsModalLogic.ts:105]: Yeah, this should be hoisted upwards, perhaps just to preflightLogic.

**Pull request merged**